ang/test/global.js

```diff
@@ -0,0 +1,9 @@
+(function (CRM) {
+  CRM.angular = { requires: {} };
+
+  /**
+   * Dependency Injection for civicase module, defined in ang/civicase.ang.php
+   * For unit testing they needs to be mentioned here
+   */
+  CRM.angular.requires['civicase'] = ['crmAttachment', 'crmUi', 'crmUtil', 'ngRoute', 'angularFileUpload', 'bw.paging', 'crmRouteBinder', 'crmResource', 'ui.bootstrap', 'uibTabsetClass', 'dialogService'];
```

Comment:
What is the usage of `CRM.angular.requires['civicase'] ` in our testing framework?

Reply:
In `https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase.js#L2`,  the dependency for `civicase` module is defined using `CRM.angRequires('civicase')`. 

The dependencies are actually mentioned in https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase.ang.php#L207. 
The backend stores the dependencies mentioned in PHP into the `CRM.angular.requires` object, which is then used by the `angRequires` function. https://github.com/civicrm/civicrm-core/blob/master/js/Common.js#L1598

This piece of code manually does that(mocks), as in unit tests backend works only in runtime.